server/dao/client.js

```diff
@@ -0,0 +1,23 @@
+"use strict";
+
+const { ObjectID } = require("bson");
+
+const clientCollectionName = "clients";
+
+// ---------------
+// GetClientByID
+// ---------------
+
+exports.getClientByID = (db, clientID) => {
```

Comment:
getClientByID is already implemented by me. in merging probably you'll need to resolve that conflict.